hikvision_tftpd.py

```diff
@@ -74,15 +75,24 @@ def _bind(self, addr):
         return sock
 
     def set_block_size(self, block_size):
+        # TODO: Don't mutate overall server for a single transfer. Use some kind of per-transfer state
         print 'Setting block size to %d' % block_size
-        self.BLOCK_SIZE = block_size
-        self._total_blocks = ((len(self._file_contents) + self.BLOCK_SIZE)
-                              // self.BLOCK_SIZE)
+        self._block_size = block_size
+        self._total_blocks = ((len(self._file_contents) + self._block_size)
+                              // self._block_size)
         if self._total_blocks > 65535:
             raise Error('File is too big to serve with %d-byte blocks.'
-                        % self.BLOCK_SIZE)
+                        % self._block_size)
         print 'Serving %d-byte %s (block size %d, %d blocks)' % (
-            len(self._file_contents), self._filename, self.BLOCK_SIZE, self._total_blocks)
+            len(self._file_contents), self._filename, self._block_size, self._total_blocks)
+
+    def set_default_block_size(self):
```

Comment:
Could you add an underscore prefix to the method name? This is Python style for private things, and this shouldn't be called outside the class.